### PR TITLE
Various fixes/improvements for 6.1

### PR DESCRIPTION
**Pull request by nmacedo**
Various improvements and small fixes, including:
- new colorblind-friendly palette (#234)
- magic color for nodes (#229)
- color of relations preserved along trace (#230)
- a warning is thrown for in-line implicit conjunctions (#212)
- meta-capabilities fixed and extended with mutable features (#174, #194)
- split view for visualizations other than graph, highlighting changes (#193)
- actual used scopes always reported (#29)
- default scopes for default runs (#144)
- fixed visualization of skolem functions with past ops (#178)
- derived functions from imported modules are now shown in the visualizer (#250)
- new theme option to automatically hide all skolem elements (#242)
- support for new electrod version with int support
- other small fixes to the visualizer (#213, #207, #206, #177, #175, #74, #233, #247, #248, #279)

### CODE REVIEW

**Comment by awwaiid** [comment]
I built this locally and have run it on a few simple examples and all looks great. I like the split panes. I've likely only exercised a fraction of these changes, but thought I'd share an experience report :)

**Comment by grayswandyr** [comment]
Nuno, should we review and merge the PR or do you still intend to work on it? 
Pinging @pkriens for eventual merge decision.

**Comment by pkriens** [comment]
I am ok to merge if Nuno is ready.

**Comment by pkriens** [comment]
@nmacedo shall I resolve this for you and merge?

File: org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/TableView.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class TableView {
     final static String  SUPERSCRIPTS = "⁰¹²³⁴⁵⁶⁷⁸⁹";
     final static String  SUBSCRIPTS   = "₀₁₂₃₄₅₆₇₈₉";
     final static String  BOX_SINGLE   = "│┌─┬┐┘┴└├┼┤";
-    final static Pattern TABLE_P      = Pattern.compile("\\s*\\{(([\\d\\w$\\s,>\"-]+))\\}\\s*");
+    final static Pattern TABLE_P      = Pattern.compile("\\s*\\{(([\\d\\w$\\s,>\"-/]+))\\}\\s*");
```
Comment (pkriens) [inline org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/TableView.java:38]: This is wrong. When you have `<char> '-' <char>` inside a `[]` block you create a range. Make sure the `-` is the last character in the block.

**Comment by nmacedo** [comment]
> @nmacedo shall I resolve this for you and merge?

done, ready for merge

**Comment by pkriens** [comment]
@nmacedo can you rebase it? Then I will merge it.

**Comment by pkriens** [comment]
I want to start the release.

**Comment by nmacedo** [comment]
> @nmacedo can you rebase it? Then I will merge it.

Done, should be ready to merge.

**Comment by awwaiid** [comment]
yaaaaasssss

**Pull request merged**